addons/mail/static/src/discuss/call/common/rtc_session_model.js

```diff
@@ -68,6 +73,53 @@ export class RtcSession extends Record {
         return this.isSelfMuted || this.isDeaf;
     }
 
+    /**
+     * @returns {MediaStream | undefined}
+     */
+    get mainVideoStream() {
+        return this.isMainVideoStreamActive && this.videoStreams.get(this.mainVideoStreamType);
+    }
```

Comment:
It looks like these changes are unrelated to SFU, it would be nice to merge them separately.

Reply:
moved to https://github.com/odoo/odoo/pull/140053